mapclassify/util.py

```diff
@@ -1,9 +1,11 @@
+import numpy as np
+
 from ._classify_API import classify as _classify
 
 
 def get_rgba(
     values,
-    classifier="quantiles",
+    scheme="quantiles",
```

Comment:
Was the `classifier` keyword part of any previous release? Or was that something new in `main` since last release?

Reply:
the whole function is new as of like a week ago

Reply:
ah yeah. whoops. LOL